apis/cloudbuild/v1alpha1/workerpool_types.go

```diff
@@ -77,6 +113,7 @@ type CloudBuildWorkerPoolStatus struct {
 	ObservedState *CloudBuildWorkerPoolObservedState `json:"observedState,omitempty"`
 }
 
+// +kcc:proto=google.devtools.cloudbuild.v1.WorkerPool
```

Comment:
I added  the comment to `status.observedState`. One worths noticing is that this changes the `status.observedState.networkConfig.peeredNetwork` to `status.observedState.networkConfig.peeredNetworkRef.external` (See the golden object log).